allensdk/brain_observatory/behavior/stimulus_processing.py

```diff
@@ -559,3 +559,67 @@ def is_change_event(stimulus_presentations: pd.DataFrame) -> pd.Series:
     is_change = is_change.fillna(False)
 
     return is_change
+
+
+def compute_trials_id_for_stimulus(
+        stim_pres_table: pd.DataFrame,
+        trials_table: pd.DataFrame) -> pd.Series:
+    """Add an id to allow for merging of the stimulus presentations
+    table with the trials table.
+
+    Parameters
+    ----------
+    stim_pres_table : pandas.DataFrame
+        Pandas stimulus table to create trials_id from.
+    trials_table : pandas.DataFrame
+        Trials table to create id from using trial start times.
+
+    Returns
+    -------
+    trials_id : pd.Series
```

Comment:
I'm wondering if you can add a note either here or where you copy the active trial ids onto the passive stimulus block, that there were no trials for the passive stimulus block, it is just to show what was the response to the same stimulus during the active block. I will definitely forget why we did this.